### PR TITLE
Subprocess Info/Example for piping

### PR DESCRIPTION
**Pull request by jeff-hykin**
After getting this answer this from the Deno discord, I figured I should do my part and share it since it's pretty basic functionality.

The one reason I think this PR might be rejected is because the example includes imports from std instead of purely using the built-in API.

### CODE REVIEW

File: examples/subprocess.md [hunk 1]
```diff
@@ -97,3 +97,52 @@ Uncaught NotFound: No such file or directory (os error 2)
...
+
+```ts
+/**
+ * subprocess_piping_to_file.ts
+ */
+
+import {
+  readableStreamFromReader,
+  writableStreamFromWriter,
+} from "https://deno.land/std@0.121.0/streams/conversion.ts";
```
Comment (bartlomieju) [inline examples/subprocess.md:113]:
```suggestion
} from "https://deno.land/std@$STD_VERSION/streams/conversion.ts";
```

File: examples/subprocess.md [hunk 2]
```diff
@@ -97,3 +97,52 @@ Uncaught NotFound: No such file or directory (os error 2)
...
+import { mergeReadableStreams } from "https://deno.land/std@0.121.0/streams/merge.ts";
```
Comment (bartlomieju) [inline examples/subprocess.md:114]:
```suggestion
import { mergeReadableStreams } from "https://deno.land/std@$STD_VERSION/streams/merge.ts";
```

**Comment by jeff-hykin** [comment]
Thanks @bartlomieju, I was wondering how pinned versions were going to work. Suggestions implemented and pushed

**Comment by lucacasonato** [comment]
Maybe this would be a good addition for https://examples.deno.land?

**Comment by jeff-hykin** [comment]
I think so 👍 

I didn't know that page existed till just now. It looks pretty handy.

**Review by kt3k — APPROVED** [review]
@jeff-hykin LGTM. Thanks for your contribution!

**Pull request merged**